src/FixedDataTableCellGroup.js

```diff
@@ -147,32 +163,21 @@ class FixedDataTableCellGroupImpl extends React.Component {
       this._staticCells[i] = this._renderCell(i, columnIndex);
     }
 
-    var style = {
-      height: props.height,
-      position: 'absolute',
-      width: props.contentWidth,
-      zIndex: props.zIndex,
-    };
-    FixedDataTableTranslateDOMPosition(
-      style,
-      -1 * props.left,
-      0,
-      this._initialRender,
-      this.props.isRTL
-    );
-
     // NOTE (pradeep): Sort the cells by column index so that they appear with the right order in the DOM (see #221)
     const sortedCells = _.sortBy(this._staticCells, (cell) =>
       _.get(cell, 'props.columnIndex', Infinity)
     );
 
+    const CellGroupComponent = this.props.shouldUseLegacyComponents
+      ? CellGroupLegacy
+      : CellGroup;
+
```

Comment:
Here we will check that either I have to use the legacy components or the new ones